templates/admin/config.tmpl

```diff
@@ -170,10 +170,19 @@
 				{{if .MailerEnabled}}
 					<dt>{{.i18n.Tr "admin.config.mailer_name"}}</dt>
 					<dd>{{.Mailer.Name}}</dd>
-					<dt>{{.i18n.Tr "admin.config.mailer_disable_helo"}}</dt>
-					<dd><i class="fa fa{{if .Mailer.DisableHelo}}-check{{end}}-square-o"></i></dd>
-					<dt>{{.i18n.Tr "admin.config.mailer_host"}}</dt>
-					<dd>{{.Mailer.Host}}</dd>
+					{{if not .Mailer.UseSendmail}}
+						<dt>{{.i18n.Tr "admin.config.mailer_disable_helo"}}</dt>
+						<dd><i class="fa fa{{if .Mailer.DisableHelo}}-check{{end}}-square-o"></i></dd>
+						<dt>{{.i18n.Tr "admin.config.mailer_host"}}</dt>
+						<dd>{{.Mailer.Host}}</dd>
+					{{else}}
+						<dt>{{.i18n.Tr "admin.config.mailer_use_sendmail"}}</dt>
+						<dd><i class="fa fa{{if .Mailer.UseSendmail}}-check{{end}}-square-o"></i></dd>
```

Comment:
The if .Mailer.UseSendmail at this line could be removed since we check it ar L173.

Reply:
Done